doc/install.txt

```diff
@@ -8,43 +8,25 @@ Installing Theano
 Requirements
 ------------
 
-In order to use Theano, the following libraries and software will need
-to be installed (MacOS and Windows users should refer to platform-specific
-instructions below for detailed installation steps):
+Theano is mainly develeopped on 64-bit Linux machines. For other architectures,
+please refer to :ref:`install_others`
```

Comment:
If I recall correctly we decided to put the links to the system-specific installation instructions directly here. 
What I would do is to get rid of the `install_others` page and copy its links here. I suggest something along these lines:

    Theano is mainly developed on 64-bit Linux machines, but we do out best to provide the prerequisites installation instruction for many common OS and architectures:
    - [CentOS](whatever is the link)
    - [Ubuntu](ubuntu prereq installation link)
    ...

    In case your favorite setup is not in this list or you are curious about them, here is the list of the prerequisites of Theano: 
    - Python
    - ...

After the list, new section "Installing Theano" with the other links